dist/stat_data.py

```diff
@@ -244,7 +244,6 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_inmem_splittable', 'in-memory page passed criteria to be split'),
     CacheStat('cache_lookaside_insert', 'lookaside table insert calls'),
     CacheStat('cache_lookaside_remove', 'lookaside table remove calls'),
-    CacheStat('cache_overflow_value', 'overflow values cached in memory', 'no_scale'),
```

Comment:
Please add a section to the upgrading docs, explaining that we removed these stats.